### DISCUSSION TITLE
Welcome to AlgoMart Discussions!

### DISCUSSION THREAD

**Discussion by smonn** [deptagency/algomart#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SkullXA** [comment]
Which wallets are currently supported?

**Reply by smonn** [reply]
Right now AlgoMart hides the "wallet" by keeping it encrypted at rest (unlockable via a passphrase). The idea is to ensure anyone can use the platform without needing to setup it up. We do have plans to support some of the major ones in the Algorand ecosystem:

- [Algorand Wallet](https://www.algorandwallet.com/)
- [MyAlgo Wallet](https://wallet.myalgo.com/)
- [AlgoSigner](https://www.purestake.com/technology/algosigner/)

But also happy to take requests and PRs!

**Reply by houssam-hamdan** [reply]
@smonn A recurring question we are getting during our investor discussions. They believe users will feel safer if they have the possibility to transfer there NFT to a non-custodial MetaMask wallet. Is this possible with AlgoMart/Algorand? If not now, could this be possible in the future?

**Comment by georgejr6** [comment]
Hey so I'm pretty new to pyteal and teal but i'm trying to deploy an nft marketplace. Where in this doc should I begin learning and what other sources should I consider?

**Reply by jakerainis** [reply]
AlgoMart doesn't actually deal with ASCs right now so no need to worry about teal or pyteal. But if you're interested in learning more about those, [Algorand's Developer Portal](https://developer.algorand.org/) has comprehensive documentation and tutorials. Also, be sure to check out [Algorand Studio](https://developer.algorand.org/articles/intro-algorand-studio-algorand-vs-code-extension/).  

As for deploying an NFT marketplace using AlgoMart without using ASCs, the main [README](https://github.com/deptagency/algomart) is a good place to start and dives deeper into the various components that the platform is comprised of.

Hope this helps!
